api/src/main/java/keywhiz/api/automation/v2/SecretDetailResponseV2.java

```diff
@@ -117,6 +136,7 @@ public SecretDetailResponseV2 build() {
   @Override public final String toString() {
     return MoreObjects.toStringHelper(this)
         .add("name", name())
+        .add("version", version())   // TODO: Should we remove this so it's not as exposed to the user?
```

Comment:
If you don't think we want to expose it to the user, then I wouldn't include it at all. We can always add it later if there does seem to be a need for it. However, if this is part of building the response, then don't we need to include the version so it can be used for choosing prior versions to roll back to? Or at least, you need some kind of unique identifier for each version to enable that feature. If we don't want to expose the real primary key of that table, you could use sequential integers for each version (sorted), or something of that nature.